source/includes/api/catalog/_product.md

```diff
@@ -391,42 +326,17 @@ curl -X "GET" "https://dashboard.shoprunback.com/api/v1/products?page=1" \
 
 ```php
 <?php
+// Load the library
+require 'path/to/lib/shoprunback-php/init.php';
 
-// Get cURL resource
-$ch = curl_init();
-
-// Set url
-curl_setopt($ch, CURLOPT_URL, 'https://dashboard.shoprunback.com/api/v1/products?page=1');
-
-// Set method
-curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'GET');
-
-// Set options
-curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-
-// Set headers
-curl_setopt($ch, CURLOPT_HTTPHEADER, [
-  "Authorization: Token token=<your token>",
- ]
-);
-
-
-// Send the request & save response to $resp
-$resp = curl_exec($ch);
-
-if(!$resp) {
-  die('Error: "' . curl_error($ch) . '" - Code: ' . curl_errno($ch));
-} else {
-  echo "Response HTTP Status Code : " . curl_getinfo($ch, CURLINFO_HTTP_CODE);
-  echo "\nResponse HTTP Body : " . $resp;
-}
+// Set your token
+\Shoprunback\RestClient::getClient()->setToken('your_token');
 
-// Close request to clear up some resources
-curl_close($ch);
-?>
+// Get all your products
+$products = \Shoprunback\Elements\Product::all();
 ```
 
-> The above command returns JSON structured like this:
+> The above command (except for PHP) returns JSON structured like this:
```

Comment:
don't talk about PHP, it's a library not a restclient, of course there is a object as response and not a JSON.